media-libs/opencv/opencv-3.4.0.recipe

```diff
@@ -249,6 +252,6 @@ INSTALL()
 
 TEST()
 {
-	cd build_haiku
-	make test
+	LIBRARY_PATH="$sourceDir/build_haiku/lib${LIBRARY_PATH:+:$LIBRARY_PATH}" \
+	make -C build_haiku test $jobArgs
```

Comment:
Any opinions on using `$jobArgs` in `TEST()` ?

(Wouldn't it be "safer" to not have it here? Or am I being too paranoid ? Or is this OK only in some cases?)

Reply:
There is way too much tests, more Than 10.000, that's why.

Reply:
OK, I see. That's a very good reason then.